foundry/quadtree.d.ts

```diff
@@ -1 +1,152 @@
-declare class Quadtree {}
+declare interface QuadTreeObject<T> {
+  /**
+   * Rectangle of this object
+   */
+  r: Rectangle;
+  /**
+   * The stored data
+   */
+  t: T;
+  /**
+   * Set of quadtrees that hold this object
+   */
+  n: Set<Quadtree<T>>;
+}
+
+declare namespace Quadtree {
+  /**
+   * Additional options which configure the Quadtree
+   */
+  interface Options {
+    /**
+     * The maximum number of objects per node
+     */
+    maxObjects?: number;
+    /**
+     * The maximum number of levels within the root Quadtree
+     */
+    maxDepth?: number;
+    /**
+     * The depth level of the sub-tree. For internal use only
+     * @internal
+     */
+    _depth?: number;
+  }
+}
```

Comment:
Just to be consistent with the other classes, this should be moved to the end of the file.